### ISSUE TITLE
[BUG] - Loading screen not clearing on login.

### ISSUE THREAD

**Issue by RimBlock** [vbawol/DayZ-Epoch#1449]
After the players character is created in game world, the loading screen does not close resulting in the player not being able to see anything but being able to move in game and others to interact with them.

What is causing it is unclear as it seems to happen quite randomly to those affected.  Thread on it here ->

What I have also noticed is that logging out (back tot he server lobby), editing character data via the backend DB and logging back in to the game does not seem to pick up the edited character data.  An example is when my test character was killed, whilst my client was in the server lobby I changed 'alive' in the character_data table to '1' and commited it.  On login to the game again, a new character was created (in 1.0.4.2, the old character would be revived).

A server reboot clears the problem untill it reoccurs.

I am speculating the player object is not being correctly cleared up in the game world in some instances but need to debug to try and confirm.

If someone could list the login / logoff script chain then I will have a look and try to debug this as I am getting it pretty consistantly on my test server.

**Comment by ian76g** [comment]
I have this problem every time.
Server admin has to kill me ingame - I use taskmanager to kill the game and then join again with a fresh character after the server admin cleared my files from the server,

How can I fix that?

**Comment by slayterson** [comment]
could the 
	progressLoadingScreen 0.8;
inside the end of the dedicated bracket cause any issues? seems out of place to me. as my ini is calling compiles between progress 4 and 5.

**Comment by ebaydayz** [comment]
Several users have reported this bug fixed after applying this change to their server:
https://github.com/vbawol/DayZ-Epoch/commit/547ba0018f8e71de641223397099da90f65f2d8d

However, we should definitely test it more for complete confirmation before closing this.

**Comment by colormeblind23** [comment]
I am still having this same bug whilst trying to get a epoch server up to speed with other epoch servers. I've debugged many of the scripts so there is no possibility of any of my other scripts causing this error. Players are able to login once and play. Second time they log into the server they get stuck at the loading screen and can hear and move around. The above change does not seem to do anything for me but break the server so that no one can join so if there is a trick or something to getting that to work for everyone please let me know or point me in the right direction if at all possible?

**Comment by ebaydayz** [comment]
It should be safe to close this issue now. It is quite old and already fixed for 1.0.6 with this commit: https://github.com/vbawol/DayZ-Epoch/commit/547ba0018f8e71de641223397099da90f65f2d8d

I just tested it. Steps to reproduce:
1. Make a fresh, stock Epoch 1.0.5.1 server with a new database and no modifications
2. Log in and load all the way into the game world. Observe that you can run around and play as normal. 
3. Disconnect and return to the server browser.
4. Without restarting the server, reconnect and attempt to load in. You will get stuck on the loading screen with a counter of 1. You will hear normal game sounds and your footsteps if you try to move around, but the loading screen will stay up. If you check your client RPT the last entry will be "PLOGIN: Login loop completed!" indicating the exitWith in compiles.sqf was executed.

Now shut down the server.
a. Copy @DayZ_Epoch1051\addons\dayz_code\init\compiles.sqf to your mission\custom folder
b. Edit compiles.sqf according to https://github.com/vbawol/DayZ-Epoch/commit/547ba0018f8e71de641223397099da90f65f2d8d
c. In mission\init.sqf replace the default compiles.sqf call with your new custom one. 
d. Restart the server with the new mission and repeat steps 2-4 above. 
e. Observe that you no longer get stuck on the loading screen at step#4. 

The people who said they still had the problem above were running custom scripts and mods. This is not the appropriate place to troubleshoot issues with third party additions. We have to focus on making stock Epoch work 100% if we are ever going to get 1.0.6. out.

tldr; please close this

**Comment by vbawol** [comment]
Rgr that, thanks!

**Comment by Evocation** [comment]
(y)
